cli/resource.go

```diff
@@ -12,13 +12,15 @@ func init() {
 	register("resource", runResource, `
 usage: flynn resource
        flynn resource add <provider>
+	   flynn resource remove <provider> <resource>
 
 Manage resources for the app.
 
 Commands:
 	With no arguments, shows a list of resources.
 
-	add   provisions a new resource for the app using <provider>.
+	add    provisions a new resource for the app using <provider>.
+	remove removes the existing <resource> provided by <provider>.
```

Comment:
Usage should include multiple spaces (at least two) between commands and their description